app/controllers/publishers_controller.rb

```diff
@@ -133,11 +133,37 @@ def update
 
   def update_unverified
     @publisher = current_publisher
-    success = @publisher.update(publisher_update_unverified_params)
-    if success
-      redirect_to(publisher_next_step_path(@publisher))
-    else
-      render(:contact_info)
+    domain = params[:publisher][:brave_publisher_id]
```

Comment:
minor: inconsistency using raw params here and strong params below with `publisher_update_unverified_params`